sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -79,6 +80,9 @@ public void define(Context context) {
       addBanditExtensions(context);
       addFlake8Extensions(context);
     }
+    if (sonarRuntime.getProduct() == SonarProduct.SONARLINT) {
```

Comment:
we also need to check sonarlint version to avoid `ClassNotFound` exceptions in older versions of SonarLint

Reply:
we'll do it via a separate PR